entrypoint-dogstatsd.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+#set -e
+
+# When using venv, activate it 1st
+if [ -n $DD_HOME ]; then
+  if [ -f "${DD_HOME}/venv/bin/activate" ]; then
+    source ${DD_HOME}/venv/bin/activate
+  fi
+fi
+
+##### Core config #####
+python /config_builder.py
+
+# ensure that the trace-agent doesn't run unless instructed to                                           
+if [ $DD_APM_ENABLED ]; then                      
+  export DD_APM_ENABLED=${DD_APM_ENABLED}         
+else                                               
+  # disable the agent when the env var is absent  
+  export DD_APM_ENABLED=false                     
+fi
```

Comment:
I didn't find a clean way to set the environment variables from python to the outside shell.
But transformed it to be way simpler here: `export DD_APM_ENABLED=${DD_APM_ENABLED:-false}`